lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -586,12 +586,12 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             cohorts_group_name = []
             if course.is_cohorted:
                 group = get_cohort(student, course_id, assign=False)
-                cohorts_group_name.append(group.name if group else '')
+                cohorts_group_name.append(unicode(group.name) if group else u'')
```

Comment:
What is `group.name` here? `unicode` or `str`? If it's the former, this is a noop, if it's the latter, this is wrong.